### PR TITLE
Vsf2 54 password reminder token fix

### PR DESCRIPTION
**Pull request by MacBalc**

### CODE REVIEW

File: tests/Application/.env [hunk 1]
```diff
@@ -31,6 +31,13 @@ JWT_PASSPHRASE=acme_plugin_development
 MAILER_URL=smtp://localhost
 ###< symfony/swiftmailer-bundle ###
 
+###> symfony/mailer ###
+# For Gmail as a transport, use: "gmail://username:password@localhost"
+# For a generic SMTP server, use: "smtp://localhost:25?encryption=&auth_mode="
+# Delivery is disabled by default via "null://null"
+MAILER_DSN=null://null
+###< symfony/mailer ###
+
```
Comment (senghe) [inline tests/Application/.env:40]: What's this? :man_shrugging:

File: tests/Behat/Context/Shop/CustomerContext.php [hunk 2]
```diff
@@ -32,18 +35,33 @@ final class CustomerContext implements Context
...
         $this->userRepository = $userRepository;
         $this->tokenFactory = $tokenFactory;
         $this->iriConverter = $iriConverter;
+        $this->entityManager = $entityManager;
+    }
+
+    /**
+     * @BeforeScenario
+     */
+    public function purgeDatabase()
```
Comment (senghe) [inline tests/Behat/Context/Shop/CustomerContext.php:59]: Is this needed? If so, could you tell me, why?

File: tests/Application/.env (hunk 1 above)
Comment (MacBalc) [reply tests/Application/.env:40]: had some problem with swift mailer so decided it would be good to move to symfony mailer anyway

Comment (MacBalc) [reply tests/Application/.env:40]: as comment at line 37 states # Delivery is disabled by default via "null://null"

File: tests/Behat/Context/Shop/CustomerContext.php (hunk 2 above)
Comment (MacBalc) [reply tests/Behat/Context/Shop/CustomerContext.php:59]: i had problems with unique emails while testing.

**Pull request merged**